wpilibj/src/main/java/edu/wpi/first/wpilibj/Tracer.java

```diff
@@ -29,6 +40,46 @@ public Tracer() {
     resetTimer();
   }
 
+  @Override
+  public void close() {
+    if (m_entryCache != null) {
+      for (IntegerPublisher pub : m_publisherCache.values()) {
```

Comment:
... These are different variables? I think you meant `m_publisherCache` in both cases.